zkevm-circuits/src/state_circuit/test.rs

```diff
@@ -23,7 +24,8 @@ use rand::SeedableRng;
 use std::collections::{BTreeSet, HashMap};
 use strum::IntoEnumIterator;
 
-const N_ROWS: usize = 1 << 16;
+const N_ROWS: usize = 1 << 16; // TODO: Uncomment
+                               // const N_ROWS: usize = 1 << 8;
```

Comment:
leftover?

Reply:
yes :P Resolved in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1046/commits/f59933089f977ec564f751c7051a824c0d5dd041